istioctl/pkg/waypoint/waypoint.go

```diff
@@ -624,7 +627,11 @@ func printWaypointStatus(ctx cli.Context, w *tabwriter.Writer, kubeClient kube.C
 			if programmed {
 				break
 			}
-			if time.Since(startTime) > waitTimeout {
+			timeoutDuration, err := time.ParseDuration(waypointTimeout)
+			if err != nil {
+				return err
+			}
+			if time.Since(startTime) > timeoutDuration {
```

Comment:
Why not apply to apply waypoint cmd?

Reply:
I didn't pay attention to the apply logic before, I have updated it, thank you